apps/web/src/shared/info/Info.styled.js

```diff
@@ -105,22 +105,13 @@ export const _visual = styled.div`
   justify-content: space-between;
 `;
 
-export const _mint = styled.div`
+export const _mint = styled(_equip)`
```

Comment:
Avoid using too many re-used components unless we standardize in design library, because if another developer changes something later without knowing there are dependencies it can cause more issues than it solves

Reply:
removed inheritance